design/20190708.certificate-request-crd.md

```diff
@@ -210,6 +408,38 @@ non-networking or other non-transient issue then the `Order` is marked as failed
 further processing will take place by the `CertificateRequest` controller on
 this resource.
 
+#### Issuing Controller
+
+Since external issuers have been built before the addition of Approved and
+Denied conditions, the issuing controller needs to be permissive. An external
+issuer may not honour an Approved condition and will sign and set a
+CertificateRequest as being Ready, before the request has been approved. The
+issuing controller must mark issuance as being successful in this case. In
+practice, this means that the issuing controller is never concerned with
+Approved conditions.
+
+External issuers that do not honour Denied conditions will sign
+CertificateRequests, even if they have a Denied condition set. In this case, the
+issuing controller will successfully complete the issuance of the Certificate.
+
+External Issuers and internal issuer that honour the Denied condition will never
```

Comment:
"Issuer" is capitalized, "issuer" is not. Does it denote a subtle difference in semantics that I may not be able to see?

Other remark: I thought we had settled on the US spelling (but we did not either write it down or say that it is only for the cert-manager codebase...)

```suggestion
External issuers and internal issuer that honor the Denied condition will never
```